temporal-testing/src/main/java/io/temporal/internal/testservice/TestWorkflowService.java

```diff
@@ -572,20 +581,46 @@ public void respondWorkflowTaskFailed(
     }
   }
 
+  private Context.CancellableContext deadlineCtx(final Duration dur) {
```

Comment:
We don't use `final` for method parameters if it's not needed (for example in case of using it in closure)